pmlc/rt/vulkan/vulkan_invocation.cc

```diff
@@ -22,6 +22,9 @@
 
 namespace pmlc::rt::vulkan {
 
+static constexpr const int kBufferCopyModeHostToDevice = 1 << 0;
+static constexpr const int kBufferCopyModeDeviceToHost = 1 << 1;
```

Comment:
There's actually a tblgen thing to make it easy to generate enum attrs.  But IIRC there was some objection to defining new attributes; definitely check w/@flaub

Reply:
No objections from my end. I think we should use tblgen for generating enums as much as we can, since it provides lots of nice extra features (stringifying/symbolizing/documentation generation, etc)

Reply:
Fixed. Thanks all.